prime-router/src/main/kotlin/cli/LivdTableDownload.kt

```diff
@@ -40,34 +42,62 @@ private const val sheetName = "LOINC Mapping"
  * for option given as below.
+ *
+ * It looks for the LIVD-SAR-CoV-2-yyyy-MM-dd.xlsx file from $cdcLOINCTestCodeMappingPageUrl.  If the file is found,
+ * it downloads the file into the ./build directory.  If not found, it will prompt error accordingly.  Next, it build
+ * the output Lookup Table (<./build/LIVD-SARS-CoV-2.csv> file) with the table name.  Finally, it updates the
+ * LIVD-SARS-CoV-2 lookup tables in the database as the new version of the table.
+ * It updates new version of the lookup table in the given --env [local, test, staging, or prod] with the default
+ * to "local" environment.
+ *
+ * Note, this command will always create new version of the lookup table.
+ *
+ * Example:
+ * The command below will download the latest LIVD mapping catalogue and create a new lookup tables as needed.
+ * It took "LIVD mapping catalogue" from the CDC website.
+ *
+ *  ./prime livd-table-download
+ *
  */
-class LivdTableDownload : CliktCommand(
+class LivdTableDownload() : CliktCommand(
     name = "livd-table-download",
     help = """
-    livd-table downloads the latest LOINC test data, so it can be ingested automatically. 
-    
-    It looks for the LIVD-SAR-CoV-2-yyyy-MM-dd.xlsx file from $cdcLOINCTestCodeMappingPageUrl.
-	If the file is found, it downloads the file into the directory specified by the --output-dir <path> option.
-	If the option is not specified, it will download the file to ./build directory.
-        
-    Example: The following command will download the latest LIVD-SARS-CoV-2-yyyy-MM-dd.xlsx from the above URL.  
-      It will store the LIVD-SARS-CoV-2-xyz.csv file under the ./junk directory.
-      
-         ./prime livd-table-download --output-file ./junk/LIVD-SARS-CoV-2-xyz.csv
+    It downloads the latest LOINC test data, extract Lookup Table, and the database as a new version. 
     """
 ) {
     private val defaultOutputDir = "./build"
-    private val outputFile by option(
-        "--output-file",
-        metavar = "<path/filename>",
-        help = "Output pathname where to store the LIVD data as CSV."
-    ).required()
+    private val outputFile = "./build/$livdSARSCov2File.csv"
+
+    /**
+     * The environment to connect to.
+     */
+    private val env by option(
+        "-e", "--env",
+        metavar = "<name>",
+        envvar = "PRIME_ENVIRONMENT",
+        help = "Connect to <name> environment.\nChoose between [local|test|staging|prod]"
+    )
+        .choice(
+            Environment.LOCAL.envName, Environment.STAGING.envName,
+            Environment.TEST.envName, Environment.PROD.envName
+        )
+        .default(Environment.LOCAL.envName, "local environment")
 
     override fun run() {
+        TermUi.echo("Downloading the lookup table ...")
+        // Download the LIVD-SARS-CoV2-yyyyMMdd.xlsx from CDC web site given above.
         val downloadedDirFile = downloadFile(defaultOutputDir)
         if (downloadedDirFile.isEmpty()) return
```

Comment:
Returning empty string on errors is a bit of an odd way to handle error situations.  While its probably OK for this code, but would not be acceptable in our "main" pipeline code.